homeassistant/components/vesync/fan.py

```diff
@@ -17,18 +17,10 @@
 )
 
 from .common import VeSyncDevice
-from .const import DOMAIN, SKU_TO_BASE_DEVICE, VS_DISCOVERY, VS_FANS
+from .const import DEV_TYPE_TO_HA, DOMAIN, SKU_TO_BASE_DEVICE, VS_DISCOVERY, VS_FANS
```

Comment:
This can be done in a preliminary PR.

Reply:
Sorry, this got caught as I was trying to split up my original code.  I'll do this when I refactor the fan module